app/models/health/exporter/health_exporter.rb

```diff
@@ -0,0 +1,222 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+require 'zip'
+require 'faker'
+require 'csv'
+module Health::Exporter
+  class HealthExporter
+    CONFIGURATION = {
```

Comment:
This is tiny, but can we make this a class method?  I worry having the bare calls to the classes in there potentially has some odd side-affects since this class and all the others get loaded at boot in production.  If it's a class method, I think we defer loading the classes in `model_class` until use.  Maybe even an instance method with caching?  At least a freeze on the end.